airbyte-integrations/connectors/source-typeform/source_typeform/manifest.yaml

```diff
@@ -0,0 +1,332 @@
+version: 0.50.0
+type: DeclarativeSource
+check:
+  type: CheckStream
+  stream_names:
+    - forms
+
+definitions:
+  items_selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path:
+        - items
+  no_selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: []
+  token_auth:
+    type: BearerAuthenticator
+    api_token: "{{ config['credentials']['access_token'] }}"
+  oauth2:
+    type: OAuthAuthenticator
+    token_refresh_endpoint: https://api.typeform.com/oauth/token
+    client_id: "{{ config['credentials']['client_id'] }}"
+    client_secret: "{{ config['credentials']['client_secret'] }}"
+    refresh_token: "{{ config['credentials']['refresh_token'] }}"
+  requester:
+    type: HttpRequester
+    url_base: https://api.typeform.com/
+    http_method: GET
+    request_parameters: {}
+    request_headers: {}
+    authenticator:
+      class_name: source_typeform.components.TypeformAuthenticator
+      token_auth: "#/definitions/token_auth"
+      oauth2: "#/definitions/oauth2"
+    request_body_json: {}
+  paginator:
+    type: DefaultPaginator
+    page_token_option:
+      type: RequestOption
+      inject_into: request_parameter
+      field_name: page
+    page_size_option:
+      type: RequestOption
+      field_name: page_size
+      inject_into: request_parameter
+    pagination_strategy:
+      type: PageIncrement
+      page_size: 200
+      start_from_page: 1
+  paginated_stream:
+    type: DeclarativeStream
+    primary_key: id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        $ref: "#/definitions/requester"
+        path: "{{ parameters.path }}"
+      record_selector:
+        $ref: "#/definitions/items_selector"
+      paginator:
+        $ref: "#/definitions/paginator"
+  trim_forms_stream:
+    $ref: "#/definitions/paginated_stream"
+    name: trim_forms
+    $parameters:
+      path: forms
+  form_id_partition_router:
+    - type: CustomPartitionRouter
+      class_name: source_typeform.components.FormIdPartitionRouter
+      parent_stream_configs:
+        - type: ParentStreamConfig
+          parent_key: id
+          partition_field: form_id
+          stream:
+            $ref: "#/definitions/trim_forms_stream"
+
+  forms_stream:
+    type: DeclarativeStream
+    name: forms
+    primary_key: id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        $ref: "#/definitions/requester"
+        path: forms/{{ stream_partition.form_id }}
+      record_selector:
+        $ref: "#/definitions/no_selector"
+      partition_router:
+        $ref: "#/definitions/form_id_partition_router"
+
+  responses_stream:
+    type: DeclarativeStream
+    name: responses
+    primary_key: response_id
+    retriever:
+      type: SimpleRetriever
```

Comment:
The connector is almost working. It is failing for a incremental test. We're going to return with more info during the week @topefolorunso